src/components/bottom-sheet/use-bottom-sheet.tsx

```diff
@@ -0,0 +1,217 @@
+import React, {
+  ReactNode,
+  RefObject,
+  useCallback,
+  useRef,
+  useState,
+} from 'react';
+import {
+  AccessibilityInfo,
+  Animated,
+  findNodeHandle,
+  LayoutChangeEvent,
+  TouchableWithoutFeedback,
+  View,
+} from 'react-native';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import {BottomSheetTexts, useTranslation} from '@atb/translations';
+
+const Backdrop = ({isOpen}: {isOpen: boolean}) => {
+  const styles = useStyles();
+  return (
+    <View
+      style={{
+        ...styles.backdrop,
+        opacity: isOpen ? 0.2 : 0,
+      }}
+      pointerEvents={'box-none'}
+    />
+  );
+};
+
+const ClickableBackground = ({
+  isOpen,
+  close,
+  height,
+}: {
+  isOpen: boolean;
+  close: () => void;
+  height: number;
+}) => {
+  const styles = useStyles();
```

Comment:
Maybe this also should be swipable with something like https://www.npmjs.com/package/react-native-swipe-gestures. Thinking adding dependency can be worth it as it also can be used for onboarding I think (or we could see how it is implemented). Right now for me it feels odd to not be able to swipe down from a handler.

Reply:
Tested this library now, but the GestureHandler did not work optimally with the ScrollView on iOS. There are some issues on the library describing the same bug. Also it does not feel smooth as there is a delay from the swipe until the sheet is closed.

My proposal is adding this without swipe gestures for now, as I think there are several higher prioritized issues.

Reply:
BottomSheet burde hekte seg inn på back button på android.
Hvis man har bottom sheet åpen og trykker tilbake, forventer brukeren at bottom sheet lukkes, men istedet navigere man til forrige skjerm.